gnomad_qc/v4/annotations/generate_freq.py

```diff
@@ -780,11 +845,12 @@ def generate_faf_grpmax(ht: hl.Table) -> hl.Table:
     faf_meta_expr[1] = faf_meta_expr[1].map(
         lambda d: hl.dict(d.items().append(("subset", "non_ukb")))
     )
+    faf_meta_expr = hl.flatten(faf_meta_expr)
     ht = ht.annotate_globals(
-        faf_meta=hl.flatten(faf_meta_expr),
-        faf_index_dict=[
-            make_faf_index_dict(hl.eval(x), label_delimiter="-") for x in faf_meta_expr
-        ],
+        faf_meta=faf_meta_expr,
+        faf_index_dict=make_freq_index_dict_from_meta(
+            faf_meta_expr, label_delimiter="-"
```

Comment:
Right now we have `_` for the freq_index_dict and `-` for the faf_index_dict, which should we use for both?